<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="colorImage">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>511</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>158</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>onStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>onStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStart()</slot>
  <slot>onStop()</slot>
 </slots>
</ui>
